python/hail/api1/dataset.py

```diff
@@ -78,15 +77,9 @@ def from_table(cls, table):
         :return: Sites-only variant dataset.
         :rtype: :py:class:`.VariantDataset`
         """
-        jvds = scala_object(Env.hail().variant, 'VariantDataset').fromKeyTable(table._jkt)
+        jvds = scala_object(Env.hail().variant, 'MatrixTable').fromKeyTable(table._jkt)
```

Comment:
I think we've learned that this could be replaced by `Env.hail().variant.MatrixTable.fromKeyTable`, right? Not asking you to change, just confirming.

Reply:
If the object has a corresponding class, I believe it generates an error.

Reply:
ahh.

Reply:
doesn't look like this is correct:
```
In [8]: Env.hail().variant.Variant.apply('1', 1, 'A', 'T')
Out[8]: JavaObject id=o25
```
There's definitely a Variant class there